app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,41 @@
+class TasksController < ApplicationController
+  
+  def index
+    @tasks = Task.all
+    render json: @tasks
+  end
+
+  def show
+    @task = Task.find(params[:id])
```

Comment:
looks good. we might clean up lines 9, 23 & 33 with a before_action.
see: http://api.rubyonrails.org/classes/AbstractController/Callbacks/ClassMethods.html